chautari-rental-service.yml

```diff
@@ -495,9 +495,7 @@ paths:
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/ResourceResponse'
+                $ref: '#/components/schemas/ResourceResponseBean'
```

Comment:
Could you please change the name of this model object?

Reply:
done.